service/src/main/resources/db/changesets/20250127_add_is_custom_value_field_to_input_defs.yaml

```diff
@@ -0,0 +1,28 @@
+# add is_custom_value field to pipeline_input_definitions table
+
+databaseChangeLog:
+  -  changeSet:
+       id:  add is_custom_value field to pipeline_input_definitions table
+       author:  mma
+       changes:
+         # add is_custom_value column to pipeline_input_definitions table
+          - addColumn:
+                tableName:  pipeline_input_definitions
+                columns:
+                    -  column:
+                        name:  is_custom_value
```

Comment:
do we want to have this be `expects_custom_value` or `is_environment_specific` or maybe something else.  I feel like i immediately misintepretted this as the value it has is a custom value (which doesnt make sense so maybe not).  I dont feel super strongly about it though

Reply:
oh i like `expects_custom_value`! i shied away from `is_environment_specific` in case there are future cases that warrant custom values that aren't necessarily env-specific.

Reply:
👍